source/libraries/HelperFunctions.ts

```diff
@@ -1,9 +1,12 @@
 import * as binascii from "binascii";
 import * as EthjsAccount from "ethjs-account";
+import * as EthjsQuery from 'ethjs-query';
 
-const DEFAULT_TEST_ACCOUNT_BALANCE = 1 * 10 ** 20; // Denominated in wei
+const DEFAULT_ETHEREUM_BLOCK_INTERVAL_MILLISECONDS: number = 1200;
```

Comment:
The default will be used when running integration tests against TestRPC, so I think we should have the interval be something like 10ms since in that case sealing should be instant (I actually don't know how TestRPC sealing works, may need to investigate but we should set it up to seal ASAP).

Reply:
Per my other comment, if this number is set too low apparently geth will start returning invalid / malformed RPC responses.  How about 500ms or 1s default?

Reply:
It feels like if Geth can't handle fist polling interval then you should use the environment variable to override.  I don't think it is a good idea to slow down our integration tests (which don't use environment variables, only the default) because Geth has a bug/odd behavior.

Reply:
Note: The way TestRPC runs work is that it checks if environment variables are set and, if so, uses them and if not it starts TestRPC and uses defaults.  This way you can just run `mocha` and have the tests "just work", without any environmental setup.

Reply:
> It feels like if Geth can't handle fist polling interval then you should use the environment variable to override. I don't think it is a good idea to slow down our integration tests (which don't use environment variables, only the default) because Geth has a bug/odd behavior.

Have you or @adrake33 run this against non-instantseal Parity?  I'd like to know if it exhibits the same behavior / how its RPC server behaves under heavy load.  Keep in mind that geth is by far the most common client (as well as the client we're currently running on our servers, and I believe it is the client used by INFURA as well) so the default settings should be configured to work correctly with geth IMO.

> Note: The way TestRPC runs work is that it checks if environment variables are set and, if so, uses them and if not it starts TestRPC and uses defaults. This way you can just run mocha and have the tests "just work", without any environmental setup.

I normally run tests using `npm test`, which can have any environment variable settings needed baked in.  However if you guys are using `mocha` by itself often I don't feel strongly about this.

Reply:
Follow-up: there isn't currently a `test` script defined in `package.json` (there are a number of other fields missing as well), and when I run `mocha` by itself I get the following error:
```
jack@substrate:~/src/augur-core$ mocha output/tests-unit/


  Universe
    1) "before each" hook for "#getTypeName()"


  0 passing (159ms)
  1 failing

  1) Universe "before each" hook for "#getTypeName()":
     Error: [ethjs-rpc]  error with payload {"id":3323123508113,"jsonrpc":"2.0","params":[],"method":"eth_accounts"} Error: [ethjs-provider-http] Invalid JSON RPC response from provider
    host: http://localhost:47622
    response:  ""
    responseURL: undefined
    status: 0
    statusText: 
  
      at node_modules/ethjs-rpc/lib/index.js:53:26
      at XMLHttpRequest.request.onreadystatechange (node_modules/ethjs-provider-http/lib/index.js:91:7)
      at XMLHttpRequestEventTarget.dispatchEvent (node_modules/xhr2/lib/xhr2.js:64:18)
      at XMLHttpRequest._setReadyState (node_modules/xhr2/lib/xhr2.js:353:12)
      at XMLHttpRequest._onHttpRequestError (node_modules/xhr2/lib/xhr2.js:542:12)
      at ClientRequest.<anonymous> (node_modules/xhr2/lib/xhr2.js:413:24)
      at Socket.socketErrorListener (_http_client.js:401:9)
      at emitErrorNT (internal/streams/destroy.js:64:8)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)
```
Do I need to have testrpc/parity/something else running / have some environmental variable(s) set to run the unit tests?

Reply:
@tinybike If there are no environment variables set, it will use testrpc by default.  Try doing it from another terminal with no environment variables set.